test/haxesharp/collections/LinqTest.hx

```diff
@@ -8,6 +8,19 @@ using haxesharp.text.StringExtensions;
 
 class LinqTest
 {
+    @Test
+    public function addAddsElement()
+    {
+        var array = ["one"];
+        array.add("two");
+        array.add("three");
+        array.add("one"); // duplicates. no worries.
+
+        Assert.that(array.length, Is.equalTo(4));
+        Assert.that(array.contains("two"));

```

Comment:
I think these `contains` asserts be checking values at specific indices, to guarantee that `add` inserting like we'd expect.

Reply:
Agreed. It's important to test/verify that it adds to the end.